pkg/driverbuilder/builder/amazonlinux.go

```diff
@@ -119,23 +119,20 @@ func script(c Config, targetType Type) (string, error) {
 	kv := kernelReleaseFromBuildConfig(c.Build)
 
 	var urls []string
-    if c.KernelUrls == nil {
-        // Check (and filter) existing kernels before continuing
-        var packages []string
-        packages, err = fetchAmazonLinuxPackagesURLs(kv, targetType)
-        if err != nil {
-            return "", err
-        }
-        urls, err = getResolvingURLs(packages)
-    } else {
-        urls, err = getResolvingURLs(c.KernelUrls)
-    }
-    if err != nil {
-        return "", err
-    }
-    if len(urls) < 2 {
```

Comment:
Dropped this. Do we really need kernel package?
In my tests, it was not needed.